packages/mask/src/social-network-adaptor/twitter.com/customization/twitter-color-schema.json

```diff
@@ -24,7 +24,7 @@
             "bottom": "#FFFFFF",
             "secondaryBottom": "rgba(255, 255, 255, 0.8)",
             "input": "#F2F6FA",
-            "modelTitleBg": "linear-gradient(180deg, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 0.9) 100%), linear-gradient(90deg, rgba(98, 152, 234, 0.2) 1.03%, rgba(98, 152, 234, 0.2) 1.04%, rgba(98, 126, 234, 0.2) 100%)",
+            "modalTitleBg": "linear-gradient(180deg, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 0.9) 100%), linear-gradient(90deg, rgba(98, 152, 234, 0.2) 1.03%, rgba(98, 152, 234, 0.2) 1.04%, rgba(98, 126, 234, 0.2) 100%)",
```

Comment:
The field name here should borrow from the UI kit. Please consult @nuanyang233.

Reply:
Confirmed, `modelTitleBg` is a typo.

Reply:
Sry, `modelTitleBg` is a  typo error.